src/app/pages/race/race.component.ts

```diff
@@ -10,7 +10,7 @@ import { Race } from '../../model/race/race';
   styleUrl: './race.component.scss',
 })
 export class RaceComponent {
-  //déclare une propriété 'aces' de type array objects 'Race'
+  //déclare une propriété 'races' de type array objects 'Race'
```

Comment:
remove dead code

Reply:
This is not a dead code, it's a commentary